horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/TestServiceImpl.java

```diff
@@ -328,7 +329,13 @@ public TestListing summary(String roles, String folder) {
          Roles.addRolesParam(identity, testQuery, 2, roles);
       }
       List<TestSummary> summaryList = testQuery.getResultList();
-
+      int count = summaryList.size();
```

Comment:
instead should be using PanacheEntityBase.count to determine the number of entities